### ISSUE TITLE
Ghosting  and other problems with a FM6124 driver

### ISSUE THREAD

**Issue by B-Zimmer** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#654]
Hello, I know this driver isn't listed, but I still thought I'd give it a try and see if I can get this working properly. I have a 64x32 panel.

![IMG_6827](https://github.com/mrcodetastic/ESP32-HUB75-MatrixPanel-DMA/assets/161541910/bb89dc9d-b35a-4ce1-bfba-190fb30e4a23)

I have read through the "READ THIS FIRST" section, I've tried changing the latch blanking and i2sspeed variables. This did seem to help, but overall ghosting was still present. I am in the process of shortening jumper wires to give it another attempt. I also have vertical output alignment issues but am in the process of fixing this. 

Furthermore, I am also seemingly unable to change brightness, but that may be a problem for another day. I can give more info or upload more photos if needed. Could this be a driver issue? A panel issue? Or something else outside my understanding?

Thank you!
Best of luck.

https://github.com/mrcodetastic/ESP32-HUB75-MatrixPanel-DMA/assets/161541910/6023f98e-6d78-47fe-a24d-0b121e140542

https://github.com/mrcodetastic/ESP32-HUB75-MatrixPanel-DMA/assets/161541910/3d49ef65-870c-46c0-8526-55371f880753

**Comment by board707** [comment]
Hi
Please show a clear photo of the panel's rear side.
Please provide a names of ALL chips on the panel - you have to find at least three types of them.

**Comment by B-Zimmer** [comment]
Yup can do. I have fixed the main issue (it was the cable length, thank you for including that in the READ ME FIRST page). However, I am still getting a split between the top and bottom halves (visible in videos). 

the three chip names are:

SM5166PS
FM6124D
74HC245KA

Thank you for the help so far

![IMG_6844](https://github.com/mrcodetastic/ESP32-HUB75-MatrixPanel-DMA/assets/161541910/35356bdb-72d0-4452-ab71-a290a7f4ea24)
![IMG_6843](https://github.com/mrcodetastic/ESP32-HUB75-MatrixPanel-DMA/assets/161541910/a582dfa4-6960-4f74-974b-1554e45ab484)

**Comment by board707** [comment]
> SM5166PS
> FM6124D

These are standard drivers, that doesn't need any specific settings to run. 
Sorry, but I have no ideas regarding your issue.

**Comment by B-Zimmer** [comment]
No worries, thank you either way.

**Comment by mrcodetastic** [comment]
Try PX matrix library and see if it works with that.
